backend/plugins/sales_api/src/modules/sales/db/models/Checklists.ts

```
@@ -0,0 +1,245 @@
+
+      //   await putChecklistActivityLog(subdomain, {
+      //     item: checklistObj,
+      //     contentType: 'checklist',
+      //     action: 'delete',
+      //   });
+
+      await checklistObj.deleteOne();
+
+      return checklistObj;
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)